### DISCUSSION TITLE
Add a recently shared projects section?

### DISCUSSION THREAD

**Discussion by GAMS2** [webdev03/scratchinfo#29] [Ideas]
Maybe you can add a recently shared projects section on an information page? It would show the three most recent user's shared projects, kind of like Scratch Stats I guess, but you can add your own spin on it.

**Comment by imdevarsh** [comment]
Cool idea!

**Reply by GAMS2** [reply]
Thanks! I hope this gets implemented! (I can try and help if you really want, but CSS and HTML is the really only thing I can do really well (and JS and Python but I'm not as good as I used to be)).

**Reply by imdevarsh** [reply]
@GAMS2 maybe this can be a widget in the you page? also right now im going to switch to my own auth system.

**Reply by GAMS2** [reply]
I saw the announcement, and, I guess so, that could be a good idea.

**Reply by imdevarsh** [reply]
@GAMS2 I'm currently working on a widgets system where this could be in the Widgets shop!

**Reply by imdevarsh** [reply]
no more widgets system now :(

**Reply by imdevarsh** [reply]
Yes it has been more than one month, and I haven't added this yet. I think it might be in the things after collabs.
